x/supplier/module/tx_stake_supplier_test.go

```diff
@@ -215,10 +262,36 @@ func TestCLI_StakeSupplier(t *testing.T) {
 				    endpoints:
 				    - publicly_exposed_url: http://pokt.network
 				      rpc_type: json_rpc
-				`, ownerAccount.Address.String(), sample.AccAddress()),
+				`, ownerAccount.Address.String(), operatorAccount.Address.String()),
 		},
 
 		// Error Paths - Service Related
+		{
+			desc:         "owner stake supplier: unauthorized valid service configs",
+			ownerAddress: ownerAccount.Address.String(),
+			config:       defaultConfig,
+			expectedErr:  types.ErrSupplierInvalidServiceConfig,
+		},
+		{
+			desc:         "owner stake supplier: missing --stake-only flag",
+			ownerAddress: ownerAccount.Address.String(),
+			config:       stakeOnlyConfig,
+			expectedErr:  types.ErrSupplierInvalidServiceConfig,
+		},
+		{
+			desc:          "operator stake supplier: missing --stake-only flag",
+			signerAddress: operatorAccount.Address.String(),
+			ownerAddress:  ownerAccount.Address.String(),
+			config:        stakeOnlyConfig,
+			expectedErr:   types.ErrSupplierInvalidServiceConfig,
```

Comment:
Feels like this should be a different error

Reply:
It's because of the way that the test case is framed. The CLI can't know what the intention was.